### DISCUSSION TITLE
Native PHP-type reported as "<missing>", issue in Zed or in phpactor?

### DISCUSSION THREAD

**Discussion by nsrosenqvist** [phpactor/phpactor#2734] [General]
Just found this message when hovering over a native type in Zed. Should this be reported in phpactor or in Zed?

<img width="486" alt="Screenshot 2024-09-03 at 10 51 29" src="https://github.com/user-attachments/assets/c469d54f-1ee5-4728-8b3e-182dc9c79805">

**Comment by dantleech** [comment]
That's a Phpactor "bug" which I've never noticed before 👍

**Reply by nsrosenqvist** [reply]
@dantleech thanks! I reported it against phpactor: https://github.com/phpactor/phpactor/issues/2735
